frontend/src/components/assessment-creation/CreateAssessementButton.tsx

```diff
@@ -1,11 +1,25 @@
 import React from "react";
 import { Button } from "@chakra-ui/react";
+import { useHistory } from "react-router-dom";
+import { CREATE_ASSESSMENT } from "../../constants/Routes";
 import { PlusOutlineIcon } from "../../assets/icons";
 
 const CreateAssessementButton = (): React.ReactElement => {
+  const history = useHistory();
+  const date = new Date();
+  const options = { month: "long", day: "numeric", year: "numeric" };
```

Comment:
```suggestion
  const options: Intl.DateTimeFormatOptions = { month: "long", day: "numeric", year: "numeric" };
```
I'm getting this error https://stackoverflow.com/questions/66590691/typescript-type-string-is-not-assignable-to-type-numeric-2-digit-in-d if type is not explicitly declared